documentation/modules/cruise-control/con-optimization-goals.adoc

```diff
@@ -59,7 +60,7 @@ Cruise Control will ignore this goal if doing so enables all the configured hard
 In Cruise Control, the following xref:master-goals[master optimization goals] are preset as hard goals:
 
 [source]
-RackAwareGoal; ReplicaCapacityGoal; DiskCapacityGoal; NetworkInboundCapacityGoal; NetworkOutboundCapacityGoal; CpuCapacityGoal
+RackAwareGoal; MinTopicLeadersPerBrokerGoal; ReplicaCapacityGoal; DiskCapacityGoal; NetworkInboundCapacityGoal; NetworkOutboundCapacityGoal; CpuCapacityGoal
```

Comment:
I assume that `MinTopicLeadersPerBrokerGoal` is also a default optimization goal?